src/ui/Base/InitializationPlaceholder.ts

```diff
@@ -314,6 +314,7 @@ export class InitializationPlaceholder {
   private getAllFacetsElements(): HTMLElement[] {
     let facetElements = $$(this.root).findAll('.CoveoFacet');
```

Comment:
Here we could use an array for all the selectors we want to catch and then build a selector from this array. Would be more concise / easier to add more selectors.